### PR TITLE
Use proper bootstrap styling for views

### PR DESCRIPTION
**Pull request by prafullbhosale**
cc @mlorbetske @Eilon for shiproom approval

### CODE REVIEW

**Comment by danroth27** [comment]
@DamianEdwards can you take a look as well?

File: src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Templates/RazorPageGenerator/Create.cshtml [hunk 1]
```diff
@@ -39,68 +39,61 @@
...
-        @:<hr />
-        @:<div asp-validation-summary="ModelOnly" class="text-danger"></div>
-    foreach (var property in Model.ModelMetadata.Properties)
-    {
-        if (property.Scaffold && !property.IsAutoGenerated && !property.IsReadOnly)
+@:<h4>@Model.ViewDataTypeShortName</h4>
+@:<hr />
+@:<div class="row">
+    @:<div class="col-md-4">
+        @:<form method="post" class="form-horizontal">
```
Comment (danroth27) [inline src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Templates/RazorPageGenerator/Create.cshtml]: Remove form-horizontal

File: src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Templates/RazorPageGenerator/Edit.cshtml [hunk 2]
```diff
@@ -39,86 +39,79 @@
...
-        @:<hr />
-        @:<div asp-validation-summary="ModelOnly" class="text-danger"></div>
-    foreach (PropertyMetadata property in Model.ModelMetadata.Properties)
-    {
-        if (property.IsPrimaryKey)
+@:<h4>@Model.ViewDataTypeShortName</h4>
+@:<hr />
+@:<div class="row">
+    @:<div class="col-md-4">
+        @:<form method="post" class="form-horizontal">
```
Comment (danroth27) [inline src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Templates/RazorPageGenerator/Edit.cshtml]: Remove form-horizontal

**Comment by prafullbhosale** [comment]
Updated. 
- Removed `form-horizontal` class
- Removed `form-actions no-color` from Delete templates.

**Pull request merged**